cadc-download-manager-server/build.gradle

```diff
@@ -22,6 +22,7 @@ def git_url = 'https://github.com/opencadc/apps'
 dependencies {
     implementation 'javax.servlet:javax.servlet-api:[3.1,)'
 
+    implementation 'org.opencadc:cadc-access-control-server:[1.3.29,1.4.0)'
```

Comment:
should not have this jar in classpath - it is server impl only

Reply:
The change uses the `OIDCUtil.ID_TOKEN_EXPIRY_MINUTES` to get the expiry.

Reply:
We could hard-code that value if the dependency is not a good idea.

Reply:
Done.